exporter/chronicleexporter/config.go

```diff
@@ -78,12 +65,6 @@ func (cfg *Config) Validate() error {
 		return errors.New("log_type is required")
 	}
 
-	if cfg.Region != "" {
```

Comment:
Do we need to check for empty endpoint here.

Reply:
I figured no because we have a default endpoint that'd get populated

Reply:
I don't know if a user puts a blank string in if that overrides it. If it doesn't then this is fine.

Reply:
I'll test